### DISCUSSION TITLE
How to deactivate the license on one computer to activate it on another?

### DISCUSSION THREAD

**Discussion by Maribloemer** [aclap-dev/video-downloadhelper#452] [License & Purchase requests]
### What's your issue?

I want to know how to deactivate the license on one computer and use it on another.
During the day I use it on my work computer and at night I use it on my personal computer. How do I make this change?

### The 10 first digits (only 10) of your license number or email address

c9bb01d3f5

### I confirm that I did not not paste my full license number

- [X] I confirm

**Comment by jcfrog** [comment]
Hi,
You don't have to move your license, you can use it on all your devices. Just validate your license key on the other computer.
